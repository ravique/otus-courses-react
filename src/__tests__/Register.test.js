import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import '../../jest.config'

import Register from "../components/pages/register";


describe('Register', () => {
    it('should render register page', () => {
        const output = shallow(
            <Register/>
        );

        expect(shallowToJson(output)).toMatchSnapshot();
    });
});