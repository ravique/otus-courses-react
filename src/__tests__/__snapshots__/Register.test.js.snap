// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register should render register page 1`] = `
<Content>
  <h2>
    Register:
  </h2>
  <RegisterForm />
</Content>
`;
